### PR TITLE
Camera polling

### PR DESCRIPTION
**Pull request by NiklasNeugebauer**
This adds a new Repeater class which handles the image polling in the base camera class.

Main benefits:

- the current rosys.on_repeat used to keep a reference to the camera. This means that cameras were not deleted once all other references (usually in the `CameraProvider`) had deleted it. This is not the case here.
- the repeat interval is now easily configurable to match the camera's frame-rate

Open questions/ improvements:

- [x] rename Repeat class
- [x] move Repeat class closer to rosys.on_repeat

### CODE REVIEW

**Review by pascalzauberzeug — CHANGES_REQUESTED** [review]

File: rosys/vision/simulated_camera/simulated_camera.py
[the remark this answers is not in the document]
Comment (NiklasNeugebauer) [reply rosys/vision/simulated_camera/simulated_camera.py:1]: I like to keep functions that are doing significant work (communicating with the camera in this case) as obvious functions. I think a property would abstract that away too much

File: rosys/rosys.py
[the remark this answers is not in the document]
Comment (NiklasNeugebauer) [reply rosys/rosys.py:1]: True, I'd like to shortly discuss with @falkoschindler if we still need the one_handler or can replace the on_repeat entirely.

[the remark this answers is not in the document]
Comment (NiklasNeugebauer) [reply rosys/rosys.py:1]: (you can join or leave your opinion here, too, of course)

**Review by falkoschindler — APPROVED** [review]

File: rosys/vision/camera/camera.py [hunk 1]
```diff
@@ -20,16 +20,15 @@ def __init__(self,
                  id: str,  # pylint: disable=redefined-builtin
                  name: Optional[str] = None,
                  connect_after_init: bool = True,
-                 streaming: bool = True,
-                 image_grab_interval: float = 0.1,
+                 streaming: bool = False,
```
Comment (rodja) [inline rosys/vision/camera/camera.py:23]: Wow. @NiklasNeugebauer, @falkoschindler this breaking change took me some time. I wondered why my camera is not delivering any images anymore.

Comment (NiklasNeugebauer) [reply rosys/vision/camera/camera.py:23]: Oh, that was unintentional. 
I will try to explicitly list all breaking changes from on, which will hopefully help me catch that.

**Pull request merged**